### PR TITLE
fail early fail well

### PR DESCRIPTION
**Pull request by eranchetz**
@innovia wdyt?

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -288,7 +288,15 @@ func setCmdLineFlags() {
 	flag.String("host", "", "Host Project ID < Required>")
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 	pflag.Parse()
+	flag.Parse()
```
Comment (innovia) [inline main.go]: whats flag.parse vs pflag.parse ?

File: main.go [hunk 2]
```diff
@@ -288,7 +288,15 @@ func setCmdLineFlags() {
...
 	viper.BindPFlags(pflag.CommandLine)
+
+	if viper.IsSet("host") {
```
Comment (innovia) [inline main.go]: shouldn't this be if not viper.IsSet ?

**Review by innovia — COMMENTED** [review]
@eranchetz  some questions...

File: main.go (hunk 1 above)
Comment (eranchetz) [reply main.go]: my mistake

File: main.go (hunk 2 above)
Comment (eranchetz) [reply main.go]: fixed

**Review by innovia — APPROVED** [review]

**Pull request merged**